GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -578,6 +581,54 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
             return base.ProcessCmdKey(ref msg, keyData);
         }
 
+        public void LoadCustomDifftools()
+        {
+            openWithDifftoolToolStripMenuItem.DropDown = null;
+            stagedOpenDifftoolToolStripMenuItem9.DropDown = null;
+            if (!AppSettings.ShowAllCustomDiffTools)
+            {
+                return;
+            }
+
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                var difftool = Module.GetEffectiveSetting(SettingKeyString.DiffToolKey);
+                var tools = await Module.GetCustomDiffMergeTools(isDiff: true);
```

Comment:
Please use explicit type instead of `var`, helps understanding the code

Reply:
ok